app/main.py

```diff
@@ -1,6 +1,38 @@
+import os
+from datetime import datetime as dt
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+BASE_URL = "https://api.weatherapi.com/v1/current.json"
```

Comment:
It's better to use a constant for the base URL as you did, but it's also recommended to use uppercase for constants' names according to the Python style guide. So, you should rename `BASE_URL` to `URL`.